tests/test_dataset.py

```diff
@@ -99,11 +99,28 @@ def make_dataset_items():
     return ds_items_with_metadata
 
 
-def test_dataset_create_and_delete(CLIENT):
+def test_dataset_create_and_delete_no_scene(CLIENT):
```

Comment:
Shouldn't you also have tests for what effect this has on upload?

Reply:
I'm testing this properly in the backend already. You think I should test it here again?

Reply:
It's both covered in the backend and here now.